src/configuration.ts

```diff
@@ -13,6 +13,7 @@ import * as path from 'path';
 import * as telemetry from './telemetry';
 
 import * as nls from 'vscode-nls';
+import { log } from 'console';
```

Comment:
I didn't bring the PR over yet for testing, still just in reading mode, to really see if this is grey or not... but it doesn't look like you're using it, correct? Unused import?

Reply:
Good catch, fixing.